pkg/api/types.go

```diff
@@ -438,6 +442,28 @@ type GlusterfsVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// RBDVolumeSource represents a Rados Block Device Mount that lasts the lifetime of a pod
+type RBDVolumeSource struct {
+	// Required: CephMonitor is the ip:port of the ceph cluster monitor
+	CephMonitor string `json:"monitor" description:"ip:port of the ceph cluster monitor"`
```

Comment:
this is the center of another topic. In gluster, we create an endpoints (i.e. array of hosts) to represent the cluster. But the endpoint concept is being debated now. 

Add @thockin @smarterclayton